cluster-autoscaler/cloudprovider/gce/templates.go

```diff
@@ -52,10 +52,15 @@ func (t *GceTemplateBuilder) getAcceleratorCount(accelerators []*gce.Accelerator
 }
 
 // BuildCapacity builds a list of resource capacities given list of hardware.
-func (t *GceTemplateBuilder) BuildCapacity(cpu int64, mem int64, accelerators []*gce.AcceleratorConfig, os OperatingSystem) (apiv1.ResourceList, error) {
+func (t *GceTemplateBuilder) BuildCapacity(cpu int64, mem int64, accelerators []*gce.AcceleratorConfig, os OperatingSystem, pods *int64) (apiv1.ResourceList, error) {
 	capacity := apiv1.ResourceList{}
 	// TODO: get a real value.
-	capacity[apiv1.ResourcePods] = *resource.NewQuantity(110, resource.DecimalSI)
+	if pods == nil {
```

Comment:
nit: maybe remove TODO above? It's pretty clear that this code no longer needs to 'get a real value' since it would now get it as a parameter.

Reply:
Done